dbt/models/iasworld/schema/iasworld.htpar.yml

```diff
@@ -30,72 +30,12 @@ sources:
             description: Application status
           - name: asmtreas
             description: Assessment reason code
-          - name: bdname1
```

Comment:
Dropping all these simply because their definitions are redundant and not useful. I've done the same with all other `name[0-9]` and `user[0-9]` columns.